pkg/project/project.go

```diff
@@ -255,6 +255,8 @@ func (p *Project) Lock() string { return p.projectfile.Lock }
 // Cache returns the cache information for this project
 func (p *Project) Cache() string { return p.projectfile.Cache }
 
+func (p *Project) IsPortable() bool { return p.projectfile.Portable && p.projectfile.Cache != "" }
```

Comment:
Why does Cache impact `IsPortable()`? I think the two are compatible.

Reply:
`state checkout org/project --portable` would end up putting a portable runtime in the local cache directory. To me that doesn't seem particularly useful. If a user did that, they might wonder what exactly that means. I feel like `--portable` is an add-on to `--runtime-path`. As a user, that's how I would see it at least. I'm not wedded to the idea though. I'll leave it up to you.

Reply:
> `state checkout org/project --portable` would end up putting a portable runtime in the local cache directory. To me that doesn't seem particularly useful. If a user did that, they might wonder what exactly that means. I feel like `--portable` is an add-on to `--runtime-path`. As a user, that's how I would see it at least. I'm not wedded to the idea though. I'll leave it up to you.

You're probably right in that most use-cases of `--portable` would probably want to be combined with `--runtime-path`, but that's a layer of flexibility that we should leave up to the user. There's no reason I can see why we would want to force the two flags together.

There could be other use-cases, eg. a runtime that doesn't like being symlinked. Not all tools are symlink aware. 

So unless there is some overhead you're worried about I'd say let's unlink them.